### DISCUSSION TITLE
GitHub pages - 404 - hugo

### DISCUSSION THREAD

**Discussion by mskuratowski** [community/community#23405] [Other Feature Feedback, Questions, & Ideas]
<p>I committed my website docs around an hour ago and it still showed 404 error.</p>
<p>I pushed the site generated by hugo.</p>
<p><a href="https://github.com/mskuratowski/mskuratowski.github.io" rel="nofollow noopener">https://github.com/mskuratowski/mskuratowski.github.io</a></p>
<p>Website is <a href="https://mskuratowski.github.io/" rel="nofollow noopener">https://mskuratowski.github.io/</a></p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/mskuratowski">@mskuratowski</a>, I am not getting a 404 on my end. The website looks awesome by the way. Please ping me again if there continues to be issues for you locally. </p>

**Comment by mskuratowski** [comment]
<p>Yeah, it works! After some time it suddenly has started working.</p>
